tools/webpack/blocks.js

```diff
@@ -220,15 +220,21 @@ module.exports = [
 		].filter( Boolean ),
 	},
 	{
+		...baseConfig,
+		watchOptions: {
+			aggregateTimeout: 200,
+		},
+		name: 'interactivity',
 		entry: {
-			// blockname: './packages/block-library/src/blockname/interactivity.js',
+			file: './packages/block-library/src/file/view/interactivity.js',
```

Comment:
This made me realize that interactive blocks are the only scripts that are transpiled with Babel from webpack. We have a two-step workflow for everything else to ensure that 3rd party projects can consume files from `build` (CJS) and `build-module` folders (ESM).